### PR TITLE
Add barometric data

### PR DESCRIPTION
**Pull request by echaidemenos**
This PR resolves https://github.com/aqualinkorg/aqualink-app/issues/779

The PR updates `metric_metrics_enum` adding 2 new columns `barometric_pressure` and `barometric_pressure_diff`. 
`barometric_pressure` is fetched from the sofar api and with it's 2 latest values a `barometric_pressure_diff` is calculated. Both values are stored in `time_series` table.

Screenshot from 2022-10-04 13-25-13

Screenshot from 2022-10-04 13-25-30
Screenshot from 2022-10-04 13-25-46
Screenshot from 2022-10-04 13-25-55

### CODE REVIEW

**Comment by ericboucher** [comment]
@echaidemenos nice work!

Would it be feasible to change the font size of the units when the page is smaller to avoid creating this big gap? What other options do we have?

Additionally, could you deploy to the programize endpoint and add some fake data for one site? Let me know if you need help with this

File: packages/website/src/common/SiteDetails/Sensor/index.tsx [hunk 1]
```diff
@@ -109,25 +116,76 @@ const Sensor = ({ depth, id, data, classes }: SensorProps) => {
...
+                  <img alt="sensor" src={sensor} />
+                </Grid>
               </Grid>
-            ))}
-          </Grid>
+            </Grid>
+
+            <Grid item xs={12}>
+              <Grid container direction="row" spacing={1}>
+                <Grid key={42} item xs={6}>
```
[the remark this answers is not in the document]
Comment (echaidemenos) [reply packages/website/src/common/SiteDetails/Sensor/index.tsx:150]: nice catch, they were left from a different attempt. Removing them, thanks.

File: packages/website/src/common/SiteDetails/Sensor/index.tsx [hunk 2]
```diff
@@ -109,25 +116,76 @@ const Sensor = ({ depth, id, data, classes }: SensorProps) => {
...
+                  <Typography
+                    className={classes.contentTextTitles}
+                    variant="subtitle2"
+                  >
+                    BAROMETRIC PRESSURE
```
[the remark this answers is not in the document]
Comment (echaidemenos) [reply packages/website/src/common/SiteDetails/Sensor/index.tsx:155]: i don't think there we use i18n for this project

File: packages/api/src/time-series/metrics.entity.ts [hunk 3]
```diff
@@ -38,13 +38,17 @@ export enum Metric {
   PRECIPITATION = 'precipitation',
   RH = 'rh',
   WIND_GUST_SPEED = 'wind_gust_speed',
+  BAROMETRIC_PRESSURE = 'barometric_pressure',
+  // BAROMETRIC_PRESSURE_DIFF is used only in latest data table and it's the difference from the previous measurement
```
Comment (ericboucher) [inline packages/api/src/time-series/metrics.entity.ts:42]: This comment us outdated I believe?

**Pull request merged**